src/lib/layers/layers-counter.component.ts

```diff
@@ -8,23 +8,34 @@ import { FaLayersComponent } from './layers.component';
   selector: 'fa-layers-counter',
   template: '',
   host: {
-    class: 'ng-fa-layers-counter',
+    class: 'fa-layers-counter',
   },
 })
-export class FaLayersCounterComponent implements OnChanges {
+export class FaLayersCounterComponent implements OnChanges, OnInit {
   @Input() content: string;
   @Input() title?: string;
   @Input() styles?: Styles;
   @Input() classes?: string[] = [];
+  @Input() position?: string;
```

Comment:
How about changing the type from `string` to the string union of `'bottom-right' | 'bottom-left' | 'top-right' | 'top-left'`? This way user will get a compilation error if they make a typo in the value and autocompletion as they pick it resulting in a better user experience.

Reply:
Ok, got it!